NextcloudTalk/CallViewController.m

```diff
@@ -533,7 +536,11 @@ - (void)setLocalVideoRect
     }
 
     UIEdgeInsets safeAreaInsets = self.view.safeAreaInsets;
-    _localVideoOriginPosition = CGPointMake(16 + safeAreaInsets.left + _collectionViewLeftConstraint.constant, 80 + safeAreaInsets.top);
+    CGSize viewSize = self.view.frame.size;
+    CGFloat defaultPadding = 16;
+    CGFloat extraPadding = 60; // Padding to not cover  participant name or mute indicator when there is only one other participant in the call
```

Comment:
I understand the reason, but it still feels a bit weird, but it might be one of those "need to get used to"-things.

Reply:
We might change it in a follow-up PR as discussed.